js/web/lib/wasm/jsep/webgpu/ops/binary-op.ts

```diff
@@ -14,6 +15,16 @@ type BinaryFunctionCall = BuiltinFunctionName|BinaryCustomExpression|{
   vector: BinaryCustomExpression;
 };
 
+const validateInputs = (inputs: readonly TensorView[]): void => {
```

Comment:
We probably need to discuss that in general should we write those `validateInputs()` functions for operator implementations in typescript. Those functions were introduced when we do webgl implementation, which is required. But now ORT registration already guards against the input length and data type, so no need to use them.